updatedmock.css

```diff
@@ -32,8 +32,8 @@
 /*here is a dummy hover example*/
 .text-1:hover {
     opacity: 1;
-    background-color: red;
-    color: green;
+    background-color: deeppink;
+    color: deepskyblue;

```

Comment:
While code reviewing (done by another person/developer) you can leave comments or a review that should be redone or discussed further.